src/main/webapp/creatives/maps/simid_maps.js

```diff
@@ -4,8 +4,51 @@ import BaseSimidCreative from '../base_simid_creative.js';
 /**
  * A sample SIMID ad that shows a map of nearby locations
```

Comment:
end with .

what is the difference between simid_maps.js and simid_map_creative.js

Reply:
I renamed the files, just forgot to push the deleted ones, fixed.